docs/developer-guide/etendo-copilot/available-tools/openapi-tool.md

```diff
@@ -0,0 +1,72 @@
+
+**Etendo API Tool**: This tool returns the OpenAPI Specification of the Etendo API, can be used by Copilot to ask for the general information of the API (endpoints, and the descriptions of the endpoints) or the specific information of the endpoints(Parameters, responses, etc). The Etendo API Tool only returns the OpenAPI Specification of the Etendo API for insert purchase orders in Etendo.
+Roughly speaking, this tool reads and returns information from the Swagger (OpenAPI Spec), allowing the Assistant to request general information from the API, for example to know what endpoints it has and what each one does. It can also request specific information about an endpoint (such as the shape of its body). This allows the helper, in real time, to investigate the structure of the API. In addition, if the API changes, the simple fact that the Swagger is updated will ensure that the wizard adapts to those changes.
+
+This Tool serves as an example and template, since to integrate with another API, as long as it has an OpenAPI Specification, a similar tool could be made but for that new specification and the assistant who has this tool, could know how to use that API.
+
+
+**API Call Tool**: This tool allows to Copilot to make a call to a API endpoint. 
+This tool will be the responsible to make the call to the API endpoint, and will return the response.
+
+## Assistant defined
+In the basic use case, a "Purchase Assistant" was implemented, in which an assistant is defined that knows the flow of how purchase orders are loaded. And it was provided with a Tool that allows to read the API specification (Etendo API Tool) and "understand" it. It was also given access to a Tool to make the API calls (APICallTool).
+
+### Configuration:
+ - In the module ```com.etendoerp.copilot.openapi.purchase``` there is a dataset with the basic configuration of the purchase assistant. It can be imported in the "Enterprise module management" window. 
+ - After import the configuration, it is necessary to config the OpenAI model for the assistant and Sync the assistant.
+ - Finally, give access to the role, configure the permissions in "Role" Window.
+!!! note
+    In the last paragraph of the prompt, the link "http://localhost:8080/etendo/?tabId=294&recordId={ORDER_HEADER_ID}" is a link for localhost, it is necessary to replace it with the real link of the Etendo system.
+
+### Other Configurations:
+- Its necessary to add the ```gradle.properties``` file with the following configuration:
+``` properties
+    ETENDO_HOST=http://localhost:8080/etendo
+```
+!!! warning
+    Replace http://localhost:8080/etendo with the real link of the Etendo system.
+
+
+## Functionality
+
+The combination of the prompt with the functional concepts, the tool that allows to read and analyze the OpenAPI Spec to see the available endpoints and the tool to make the API calls, allows the defined assistant to load the purchase and its lines.
+
+## How to integrate Copilot with other API's
```

Comment:
Agregar capturas de pantalla del swager y ejemplos de código

Reply:
![image](https://github.com/etendosoftware/docs/assets/106354215/e3742b7f-81c3-4c70-83c2-bf02202a0cad)
If you like, we can ask Ale to design a graphic that conceptualizes the operation of the api call tool and the whole architecture.

Reply:
I will add an example for the petStore then, with code snippets to at least guide the process a bit. Regarding Ale's graph, I'm fine with it, but not to wait for it, could we leave it as an improvement outside the task? Let's do as you see fit.